airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -99,26 +136,34 @@ def get_incremental_stream_config(google_api: GoogleAds, config: Mapping[str, An
         )
         return incremental_stream_config
 
-    def get_all_accounts(self, google_api: GoogleAds, customers: List[CustomerModel], customer_status_filter: List[str]) -> List[str]:
+    @staticmethod
+    def get_all_accounts(google_api: GoogleAds,
+                         customers: List[CustomerModel],
+                         customer_status_filter: List[str]) -> Iterable[Mapping[str, Any]]:
         customer_clients_stream = CustomerClient(api=google_api, customers=customers, customer_status_filter=customer_status_filter)
-        for slice in customer_clients_stream.stream_slices():
-            for record in customer_clients_stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice=slice):
+        for stream_slice in customer_clients_stream.stream_slices():
+            for record in customer_clients_stream.read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slice):
                 yield record
 
     def _get_all_connected_accounts(
-        self, google_api: GoogleAds, customer_status_filter: List[str]
-    ) -> Iterable[Iterable[Mapping[str, Any]]]:
-        customer_ids = [customer_id for customer_id in google_api.get_accessible_accounts()]
+        self, google_api: GoogleAds,
+            customer_status_filter: List[str],
+            manager_ids: List[str]
+    ) -> Iterable[Mapping[str, Any]]:
+        customer_ids = [customer_id for customer_id in google_api.get_accessible_accounts()
```

Comment:
Would you mind updating the description in spec.json to state that if no manager ID is present we'll get all customers for all accounts? (This can be done in the separate PR if we move the manager ID logic there.)